homeassistant/components/media_player/sonos.py

```diff
@@ -608,6 +613,13 @@ def update(self):
             else:
                 playlist_size = int(playlist_size)
 
+            if playlist_size is None or playlist_size == 0:
```

Comment:
There is not necessarily any added value I think. I figured this would be a bit more clean, as this way the logic for when play/pause/etc is supported is explicit. Also I wasn't not sure where I would best implement the catch statements. I guess that would be in the `self.media_pause()` (etc) functions?